tools/__tasks__/compile/fonts/index.config.js

```diff
@@ -1,973 +1,507 @@
+                    'GuardianTextEgyptian/noalts-not-hinted/GuardianTextEgyptian-Bold.woff2',
             },
         ],
     },
+    // *** Guardian Text Egyptian - woff - Not Hinted ***
     {
-        dest: 'GuardianTextSansWebCleartypeHinted.woff.json',
-        'font-family': '"Guardian Text Sans Web"',
+        dest: 'GuardianTextEgyptian.woff.json',
+        'font-family': '"Guardian Text Egyptian"',
         fonts: [
             {
                 src:
-                    'hinting-cleartype/kerning-on/original/GuardianTextSansWeb/GuardianTextSansWeb-Regular.woff',
+                    'GuardianTextEgyptian/noalts-not-hinted/GuardianTextEgyptian-Regular.woff',
             },
             {
                 'font-style': 'italic',
                 src:
-                    'hinting-cleartype/kerning-on/ascii/GuardianTextSansWeb/GuardianTextSansWeb-RegularItalic.woff',
+                    'GuardianTextEgyptian/noalts-not-hinted/GuardianTextEgyptian-RegularItalic.woff',
             },
             {
                 'font-weight': '700',
+                'font-style': 'italic',
                 src:
-                    'hinting-cleartype/kerning-on/original/GuardianTextSansWeb/GuardianTextSansWeb-Medium.woff',
+                    'GuardianTextEgyptian/noalts-not-hinted/GuardianTextEgyptian-Bold.woff',
             },
         ],
     },
+    // *** Guardian Text Egyptian - ttf - Not Hinted ***
     {
-        dest: 'GuardianTextSansWebCleartypeHinted.ttf.json',
-        'font-family': '"Guardian Text Sans Web"',
+        dest: 'GuardianTextEgyptian.ttf.json',
+        'font-family': '"Guardian Text Egyptian"',
         fonts: [
             {
                 src:
-                    'hinting-cleartype/kerning-on/original/GuardianTextSansWeb/GuardianTextSansWeb-Regular.ttf',
+                    'GuardianTextEgyptian/noalts-not-hinted/GuardianTextEgyptian-Regular.ttf',
             },
             {
                 'font-style': 'italic',
                 src:
-                    'hinting-cleartype/kerning-on/ascii/GuardianTextSansWeb/GuardianTextSansWeb-RegularItalic.ttf',
+                    'GuardianTextEgyptian/noalts-not-hinted/GuardianTextEgyptian-RegularItalic.ttf',
             },
             {
                 'font-weight': '700',
+                'font-style': 'italic',
                 src:
-                    'hinting-cleartype/kerning-on/original/GuardianTextSansWeb/GuardianTextSansWeb-Medium.ttf',
+                    'GuardianTextEgyptian/noalts-not-hinted/GuardianTextEgyptian-Bold.ttf',
             },
         ],
     },
+    // *** Guardian Text Egyptian - woff2 - Hinted ***
     {
-        dest: 'GuardianTextEgyptianWebAutoHinted.woff2.json',
-        'font-family': '"Guardian Text Egyptian Web"',
+        dest: 'GuardianTextEgyptianHinted.woff2.json',
+        'font-family': '"Guardian Text Egyptian"',
         fonts: [
             {
                 src:
```

Comment:
Regular weight should always be 400, I think. 300 is for Light. Also weight numbers are missing for Regular weights above for all flavours but ttf hinted.